include/taskolib/Tag.h

```diff
@@ -0,0 +1,137 @@
+/**
+ * \file   Tag.h
+ * \author Lars Fröhlich
+ * \date   Created on July 17, 2024
+ * \brief  Declaration of the Tag class.
+ *
+ * \copyright Copyright 2024 Deutsches Elektronen-Synchrotron (DESY), Hamburg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 2.1 of the license, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#ifndef TASKOLIB_TAG_H_
+#define TASKOLIB_TAG_H_
+
+#include <iosfwd>
```

Comment:
Not sure what this is for

Reply:
That contains a forward declaration for `std::ostream`. It is basically a lightweight alternative to `<iostream>`.